docs/faq.rst

```diff
@@ -987,6 +987,12 @@ stable. If the directory where you mount a filesystem is different every time,
 Borg assumes they are different files. This is true even if you backup these
 files with relative pathnames - borg uses full pathnames in files cache regardless.
 
+It is possible for some filesystems, such as mergerfs_,
+to return inconsistent inode numbers across runs, causing borg to consider them
+changed. A workaround is to set the option ``--ignore-inode`` so that files with
```

Comment:
iirc we deprecated --ignore-inode in favour of --files-cache (at least for master branch, please also check 1.1-maint, see docs/changes.rst.

Reply:
also it could be noted that unstable inodes happen also on network filesystems.

Reply:
I've updated that, --ignore-inode was indeed deprecated back in 1.1.0rc4 so the doc changes should apply to the maintenance branch as well.